graphql-server/src/config.ts

```diff
@@ -11,6 +11,7 @@ const config = {
   apiPort: process.env.API_PORT || 5000,
   remoteSchemaConfig: remoteSchemas,
   ethMainnetProvider: process.env.ETH_MAINNET_PROVIDER as string,
+  bscProvider: 'https://bsc-dataseed.binance.org/',
```

Comment:
Should this be in .env or will this always be constant?

Reply:
constant, it's provided by binance themselves